hazelcast/src/main/java/com/hazelcast/config/alto/AltoConfig.java

```diff
@@ -0,0 +1,39 @@
+package com.hazelcast.config.alto;
+
+import com.hazelcast.config.InvalidConfigurationException;
+
+public class AltoConfig {
+    private static final boolean DEFAULT_ENABLED = false;
+    private static final int DEFAULT_EVENTLOOP_COUNT = Runtime.getRuntime().availableProcessors();
+
+    private boolean enabled = DEFAULT_ENABLED;
```

Comment:
For now this is ok, but in the future we want to read this from System properties/Hazelcast properties.